DESCRIPTION

```diff
@@ -48,6 +48,7 @@ Imports:
     ellipsis,
     glue,
     igraph,
+    jsonlite,
```

Comment:
We're close to 20 imports, should we make this optional?

Reply:
Done in https://github.com/cynkra/dm/pull/918/commits/58ec7a5bb3f6bbbd98fbc7ed4128d927ffde8d81